### PR TITLE
Refactor Covid19screen <FormResult> to use composition and lookups

### PR DESCRIPTION
**Pull request by mchelen-gov**
- [x] use composition to deduplicate markup into `<Complete>` component
- [x] use lookup and ternary operators to simplify conditional logic

### CODE REVIEW

File: src/applications/coronavirus-screener/components/FormResult.jsx [hunk 1]
```diff
@@ -51,27 +17,79 @@ export default function FormResult({ formState }) {
...
-  } else {
-    result = pass;
-    resultClass = 'pass';
-    recordScreeningToolEvent('Pass');
+
+  const Incomplete = () => <div>Please answer all the questions above.</div>;
+
+  const Pass = () => (
+    <Complete>
+      <i aria-hidden="true" role="presentation" className="fas fa-check" />
```
[the remark this answers is not in the document]
Comment (mchelen-gov) [reply src/applications/coronavirus-screener/components/FormResult.jsx:25]: icon is decorative

**Review by stephenBarrs — APPROVED** [review]
LGTM

File: src/applications/coronavirus-screener/components/FormResult.jsx [hunk 2]
```diff
@@ -51,27 +17,79 @@ export default function FormResult({ formState }) {
...
+      <h2 className="vads-u-font-size--h1">OK to proceed</h2>
+    </Complete>
+  );
+
+  const Fail = () => (
+    <h2 className="vads-u-font-size--h1">More screening needed</h2>
```
Comment (stephenBarrs) [inline src/applications/coronavirus-screener/components/FormResult.jsx:31]: Need to wrap this with <Complete>

Comment (mchelen-gov) [reply src/applications/coronavirus-screener/components/FormResult.jsx]: good catch!

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**